lab-brianna/tests/tests.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

You have to rename your folder to "test" so that I can run mocha on it.